cfme/containers/topology.py

```diff
@@ -0,0 +1,9 @@
+from cfme.common import TopologyMixin
+from cfme.fixtures import pytest_selenium as sel
+
+
+class ContainerTopology(TopologyMixin):
```

Comment:
@gshefer @psav I just created a new PR for this class. 

Included this classmethod, but replaced the navigation with a proper navmazing destination.

https://github.com/ManageIQ/integration_tests/pull/3758